style.css

```diff
@@ -0,0 +1,212 @@
+/*===========================================
+      TABLE OF CONTENTS
+=======================================
+
+1. NAVBAR
+2. FORM BODY
+3. FOOTER
+
+*/
+
+@import url("https://fonts.googleapis.com/css?family=Montserrat&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Nunito&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Nunito", sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+/*============================================
+      NAVBAR
+=============================================*/
+nav {
+  background: #eceef1;
+  display: flex;
+  justify-content: flex-end;
+  padding: 10px 30px;
+}
+
+nav button {
+  background: #0077c5;
+  border-radius: 4px;
+  border: 1px solid #4c4c4c;
+  width: 102.36px;
+  height: 34px;
+  color: #ffffff;
+  font-size: 13px;
+}
+
+nav p {
+  font-size: 12px;
+  color: #505050;
+  padding: 3px 0px;
```

Comment:
Correct the linter checks, for example here `Unexpected unit (length-zero-no-unit) [error]` is telling you that 0 doesn't need units (px, rem, etc)